specs/proposals/blob-signing.md

```diff
@@ -0,0 +1,335 @@
+
+- **Detailed command usage**, which is covered in the individual command specifications at [Notation CLI specs](https://github.com/notaryproject/notation/tree/main/specs/commandline).  
+- **Distribution of certificates** for verification in different scenarios.  
+- **Blob signature definitions**, as outlined in the [Notary Project specification](https://github.com/notaryproject/specifications/blob/main/specs/signature-specification.md#blob-payload).  
+- **CI/CD integration**, as this document specifically focuses on using Notation CLI commands.  
+
+In general, all blob-related commands are grouped under the `notation blob` command per the [blob command spec](https://github.com/notaryproject/notation/blob/main/specs/commandline/blob.md).
+
+### Blob Sign and verify with file-based distribution
+
+For file-based distribution, such as SBOMs or release artifacts shared via a website or file transfer system, the process of signing and verifying blobs follows a similar flow. The steps below outline the user experience for signing and verifying a file, using an SBOM file named `sbom.json` as an example.
+
+**Prerequisites:**
+- SBOMs, release artifacts or other files are created and ready for publishing
+
+**Steps:**
+
+1. Sign a file on filesystem with a specified signature envelope:
+
+    ```shell
+    notation blob sign --id myKeyId --signature-format "cose" --media-type "application/spdx+json" sbom.json
+    ```
+    This command generates a signature file named `sbom.json.cose.sig` in the current working directory. The file name follows the [Notary Project specification](https://github.com/notaryproject/specifications/blob/main/specs/signature-specification.md#blob-signatures). The signature format is **COSE**, as specified by the `--signature-format` flag. The default format is **JWS**. The `--media-type` flag specifies the **media type** of the blob. In this example, the content of `sbom.json` is in the format `application/spdx+json`. This flag is **optional**. If omitted, the default media type **`application/octet-stream`** is used.  
+
+2. Publish both the file and corresponding signature using any file transfering mechanism. Both the file and its signature can be packaged into one file (e.g., a tarball) for transferring.
+
+3. Download or fetch both the file and signature. If they are packaged into one file, for example, a tarball, unpackage the file to get separate file and signature.
+
+4. Verify the file against the signature:
+
+   - Add the root CA certificate and set up the trust store (The same experience for verifying container images):
+
+        ```shell
+        notation cert add --type ca --store myCACerts root.crt
+        ```
+
+     Confirm the certificate is added.
+
+        ```shell
+        notation cert ls
+        ```
+
+        > [!NOTE]
+        > Learn more options using `notation cert ls --help`.
```

Comment:
nit: It seems those two lines are not rendered well on GitHub in a list. We probably need to reformat it for better rendering.